__tests__/api.test.js

```diff
@@ -109,3 +109,47 @@ describe("api/articles", () => {
     });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  it("should return a status code of 200 and an array", async () => {
+    const response = await request(app)
+      .get("/api/articles/1/comments")
+      .expect(200);
+  });
+  it("should return an array of comment objects with the correct properties", async () => {
+    const { body } = await request(app)
+      .get("/api/articles/1/comments")
+      .expect(200);
```

Comment:
In case there are no comments, this test will still pass, so please check the length of the array before the forEach to make sure it contains the comments.